client/network/src/service.rs

```diff
@@ -659,6 +715,8 @@ pub struct NetworkWorker<B: BlockT + 'static, S: NetworkSpecialization<B>, H: Ex
 	from_worker: mpsc::UnboundedReceiver<ServerToWorkerMsg<B, S>>,
 	/// Receiver for queries from the light client that must be processed.
 	light_client_rqs: Option<mpsc::UnboundedReceiver<RequestData<B>>>,
+	/// Senders for events that happen on the network.
+	events_streams: Vec<mpsc::UnboundedSender<Event>>,
```

Comment:
```suggestion
	event_streams: Vec<mpsc::UnboundedSender<Event>>,
```

I would call this `event_streams` given that a single `stream` already implies the plural for `event`.